tfhe/src/shortint/server_key/tests/modulus_switch_compression.rs

```diff
@@ -0,0 +1,45 @@
+use super::NB_TESTS;
+use crate::shortint::keycache::KEY_CACHE;
+use crate::shortint::parameters::*;
+use crate::shortint::server_key::tests::parametrized_test::create_parametrized_test;
+use rand::Rng;
+
+create_parametrized_test!(shortint_modulus_switch_compression);
```

Comment:
this will fail with multi bit right ? so we need to use the version where we choose which parameters are tested

like

```rust
create_parametrized_test!({
    PARAM_SET1,
    PARAM_SET2,
});
```